benefits/templates/admin/agency-dashboard-index.html

```diff
@@ -0,0 +1,78 @@
+{% extends "admin/base.html" %}
+{% load i18n static %}
+
+{% block extrastyle %}
+    <style>
+        html[data-theme="light"], :root {
+          --primary: #045b86;
+          --secondary: #045b86;
+          --link-fg: #ffffff;
+          --body-quiet-color: #000000;
+        }
+        html[data-theme="dark"], :root {
+            --primary: #045b86;
+            --secondary: #045b86;
+            --link-fg: #ffffff;
+            --body-quiet-color: #000000;
+          }
+    </style>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700" rel="stylesheet" type="text/css">
+    <link href="https://california.azureedge.net/cdt/statetemplate/6.0.7/css/cagov.core.min.css" rel="stylesheet">
+    <link href="{% static "css/styles.css" %}" rel="stylesheet">
+    <link href="{% static "img/favicon.ico" %}" rel="icon" type="image/x-icon" />
+{% endblock extrastyle %}
+
+{% block title %}
+    Agency Dashboard
+{% endblock title %}
+
+{% block branding %}
+    <div id="header-container" class="navbar navbar-expand-sm navbar-dark bg-primary justify-content-between">
+        <div class="container">
+            <span class="navbar-brand p-0">
+                <img class="sm d-lg-none" src="{% static "img/logo-sm.svg" %}" width="90" height="51.3" alt="{% translate "California Integrated Travel Project: Benefits logo (small)" context "image alt text" %}" />
+                <img class="lg d-none d-lg-block" src="{% static "img/logo-lg.svg" %}" width="220" height="50" alt="{% translate "California Integrated Travel Project: Benefits logo (large)" context "image alt text" %}" />
+            </span>
+        </div>
+        <h1 class="p-0">Administrator</h1>
+    </div>
+{% endblock branding %}
+
+{% block coltype %}
+    w-100
+{% endblock coltype %}
+
+{% block bodyclass %}
+    {{ block.super }} dashboard
+{% endblock bodyclass %}
+
+{% block nav-breadcrumbs %}
+{% endblock nav-breadcrumbs %}
+
+{% block nav-sidebar %}
+{% endblock nav-sidebar %}
+
+{% block content_title %}
+{% endblock content_title %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-lg-12">
+            <h1 class="pt-0 pb-3 text-left">{{ agency.long_name }}</h1>
+            <div class="border border-3 p-3">
+                <h2 class="pt-0 pb-2 text-left">In-person enrollment</h2>
+                <p>Verify transit benefit eligibility using agency criteria and complete a rider's open-loop card enrollment.</p>
```

Comment:
```suggestion
                <p>Verify transit benefit eligibility using agency criteria and complete a rider’s open-loop card enrollment.</p>
```

Reply:
Just this small thing, otherwise this PR LGTM.

There are some questions about how we approach this Header section for example across all the new pages, but I think that kind of refactoring can happen later when we need it. And that for now, this is a good start.

Reply:
> There are some questions about how we approach this Header section for example across all the new pages, but I think that kind of refactoring can happen later when we need it. 

Thanks for noting this; I think this is an important question to answer sooner than later. 

@lalver1 and I talked more about his idea from https://github.com/cal-itp/benefits/pull/2313#issuecomment-2307464485 when we paired on Friday, and the plan is for him to open a separate PR that extracts out a base template that all the new pages can use.

Reply:
Here is the full plan: https://github.com/cal-itp/benefits/issues/2246#issuecomment-2310896594

It is the same as what @lalver1 wrote in this PR's description.
> Another PR will implement the top right corner "user tools" panel for transit agency staff, and another PR will implement a base html template that will be extended by all templates in the in_person app.